src/components/Inputs/NumberInput/NumberInput.md

```diff
@@ -11,6 +11,9 @@ const formChangeHandlerStub = () => {}
   placeholder='number input'
   formChangeHandler={formChangeHandlerStub}
   validator={(n) => {
+    if (n > Number.MAX_SAFE_INTEGER) {
```

Comment:
lol

Reply:
yeah, this was boggling me like "how come the validator stops working after a certain number of digits?" then I'm like, oh yeah, no biginty int 😋